.drone.yml

```diff
@@ -390,3 +397,26 @@ matrix:
       REPO_BRANCH: master
       REPO_PATH: .
       MODE: MAKE
+
+    - REPO_NAME: windows_phone
+      REPO_URL: https://github.com/owncloud/OwncloudUniversal.git
+      REPO_GIT: git@github.com:owncloud/OwncloudUniversal.git
+      REPO_BRANCH: master
+      REPO_PATH: .
+      MODE: MAKE
+
+    - REPO_NAME: client
+      REPO_URL: https://github.com/owncloud/client.git
+      REPO_GIT: git@github.com:owncloud/client.git
+      REPO_BRANCH: tx
+      REPO_PATH: translations
+      MODE: MAKE
+      TRANSLATION_READER_IMAGE: rabits/qt:5.12-desktop
+
+    - REPO_NAME: client-nsis
+      REPO_URL: https://github.com/owncloud/client.git
+      REPO_GIT: git@github.com:owncloud/client.git
+      REPO_BRANCH: tx
+      REPO_PATH: admin/win/nsi/l10n
+      MODE: MAKE
+      TRANSLATION_WRITER_IMAGE: python:2.7-stretch
```

Comment:
The transifex image works here, tx is a python tool.

Reply:
It does not :-P 

It's missing the polib python package and it does not have pip, so I can't install it. We could probably add python3-polib to the transifex image tho.

Reply:
Ah, it has pip3 .. 

I will check the script actually works with python3 and change to the transifex image in that case.

Reply:
It does not work with python3. It's not completely trivial to fix and I don't think it's worth messing with that  now, as it's legacy and deprecated anyway. It's only still there for the community to be able to build an installer. Officially supported is only MSI.

Reply:
To be clear: The issue is not the transifex tool alone, but more importantly https://github.com/owncloud/client/blob/v2.5.4/admin/win/nsi/l10n/bin/build_locale_nsi.py .

Reply:
So just replace python3 with python2 within the transifex image? It doesn'T matter if it provides python2 or python3 from the perspective of Transifex.

Reply:
We need the TRANSLATION_WRITER_IMAGE anyway for qt tools, so we can also use a python image here. Changing the transifex image back to python2 feels wrong... we might need python3 in the future. The current implementation seems most resilient to me.